protocols/session/src/socket/ack_state.rs

```diff
@@ -469,7 +469,7 @@ impl<const C: usize> SocketState<C> for AcknowledgementState<C> {
             .into_iter()
             .try_for_each(|s| {
                 tracing::trace!(seg_id = %s.id(), "retransmit segment on request");
-                ctx.ctl_tx.unbounded_send(SessionMessage::Segment(s))
+                ctx.ctl_tx.try_send(SessionMessage::Segment(s))
```

Comment:
Error result should be handled and logged.

Reply:
It is mapped in the next step.